public/js/reducers/sources.js

```diff
@@ -43,14 +43,11 @@ function update(state = State(), action: Action) : Record<SourcesState> {
       return state.mergeIn(["sources", action.source.id], source);
     }
 
-    case "LOAD_SOURCE_MAP":
-      if (action.status == "done") {
-        return state.mergeIn(
-          ["sourceMaps", action.source.id],
-          action.value.sourceMap
-        );
-      }
-      break;
+    case "SOURCE_MAP_LOADED":
+      return state.setIn(
+        ["sourceMaps", action.source.id],
+        action.map
+      );
```

Comment:
We actually do that. The redux state isn't used for anything. I suppose for not we could just kill this code and not store it in redux for the time being.